### PR TITLE
Remove placeholders

### PR DESCRIPTION
**Pull request by callum**
Gets rid of placeholders altogether as per our chat in #frontend.

A nice upside is that patterns can now have arguments. The expand click area one could have parameters to tell it how much to expand by, for example.

I'm rubbish at semver... What version should this be? By convention it seems like it would be `1.0.0` because this isn't backwards compatible, but I'm not really sure.

cc @ninjabiscuit @cjbell @raffij

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "node_modules/.bin/mocha --reporter spec --recursive"
+    "test": "mocha -R spec --recursive",
```
Comment (ninjabiscuit) [inline package.json]: How come you removed the reference to `node_modules/.bin/mocha`? Now dependencies have to be installed globally in order to work correctly

**Comment by ninjabiscuit** [comment]
I think 0.2.0 makes sense. Good job dude, looks good! :+1:

File: package.json (hunk 1 above)
Comment (callum) [inline package.json]: https://docs.npmjs.com/misc/scripts#path

Comment (ninjabiscuit) [inline package.json]: I stand corrected. Carry on

**Comment by callum** [comment]
Can you add me as an owner @ninjabiscuit so I can publish it?

https://docs.npmjs.com/cli/owner

**Comment by ninjabiscuit** [comment]
Sure thing. Can you bump the version to 0.2.0?

**Comment by callum** [comment]
Yup I'll sort all that

**Comment by cjbell** [comment]
LGTM! `@include` all the things

**Pull request merged**